code/modules/admin/verbs/adminhelp.dm

```diff
@@ -209,6 +211,9 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	return ..()
 
 /datum/admin_help/proc/AddInteraction(formatted_message)
+	if(heard_by_no_admins && usr.client != initiator)

```

Comment:
usr is a hack I hate you

Reply:
You got a better idea, sir?

Reply:
All this check is is literally just to see someone who is not the initiator has done something with the ticket